### PR TITLE
Adds missing package json fields

### LINKED ISSUE #359: Get rid of distracting warning message from `t2 init` created script
➜  blinky  npm install accel-mma84
npm WARN package.json blinky@0.0.0 No repository field.
npm WARN package.json blinky@0.0.0 No README data
accel-mma84@0.2.4 node_modules/accel-mma84
└── sync-queue@0.0.1

### PR DESCRIPTION
**Pull request by johnnyman727**
Fixes #359.

@wprater easy code review :)

### CODE REVIEW

**Comment by LinusU** [comment]
You don't need to add a `readme` field, it's populated automatically from a `README.md` file. I think it's better to also generate that file when scaffolding a new project?

**Comment by johnnyman727** [comment]
@LinusU We could add a README.md to `t2 init` but as it is, it just copies `/resources/index.js` to the directory and has some options for creating a package.json. It's not an actual node module or anything.

File: resources/init-default.js [hunk 1]
```diff
@@ -159,5 +159,7 @@ module.exports = {
     "email": config['init.author.email'],
     "url": config['init.author.url']
   } : undefined,
-  "license": 'MIT'
+  "license": 'MIT',
+  "repository": "www.github.com/tessel/t2-cli",
```
Comment (rwaldron) [inline resources/init-default.js]: 0_o This is a duplicate https://github.com/tessel/t2-cli/blob/jon-fix-init-pkg-json/resources/init-default.js#L134-L154

Comment (johnnyman727) [inline resources/init-default.js]: Nice catch! Will make a quick fix.

File: resources/init-default.js [hunk 2]
```diff
@@ -135,7 +135,7 @@ module.exports = {
     try {
       var gitConfig = fs.readFileSync('.git/config', 'utf8');
     } catch (e) {
-      gitConfig = null;
```
Comment (rwaldron) [inline resources/init-default.js]: Ah-ha!

**Pull request merged**